modules/Emails/javascript/Email.js

```diff
@@ -214,6 +214,11 @@ function multiFiles( list_target){
                 var url = 'index.php?to_pdf=1&module=EmailTemplates&action=AttachFiles',
                     new_element;
 
+                // allow attachments with TinyMCE or Direct HTML;
+                if (typeof mozaik == 'undefined') {
+                    var mozaik={};
```

Comment:
at line 225 "`if(mozaik && mozaik.uploadPathField) ...`"  evaluating `mozaik` as boolean and testing in the `if` condition is unnecessary anymore..